tests/Makefile

```diff
@@ -1,12 +1,13 @@
-PATH := ../deps/luajit/usr/local/bin:$(PATH)
+PATH := ../deps/luajit/usr/local/bin:../../deps/luajit/usr/local/bin:$(PATH)
```

Comment:
For running lua tests, what about requiring them to have a suffix of ".lua"?  Then we can run them directly if we want, because of #!/usr/bin/env luajit, but when running from the Makefile, the Makefile can invoke them with the luajit from the pflua build.